vault/post_install.md

```diff
@@ -0,0 +1,135 @@
+## Hashicorp Vault
```

Comment:
the post_install looks quite big 
```A final optional file is the post_install.md which the user will be able to view in the web UI to see additional instructions on how to use the application. It's recommended to keep this short and succinct, if you need to manage LOTS of content, then please create a community learn guide and link to it from within the post_install.md```

Reply:
I removed the manual instructions and left the link to the official documentation where the process is also described.

Reply:
great  , but after it's live, it would be good a create a short guide on how to use it and then link back to the post_install